src/api/app/controllers/webui/project_controller.rb

```diff
@@ -72,15 +72,11 @@ def autocomplete_incidents
   end
 
   def autocomplete_packages
-    packages = []
-    packages = @project.packages.autocomplete(params[:term]).pluck(:name) if @project
```

Comment:
Yes, for sure. this action is behind a before_filter called **set_project** that ensures that...
```
  before_filter :set_project, only: [:autocomplete_packages, :autocomplete_repositories, :users, :subprojects,
```